test/xcode-specs.js

```diff
@@ -83,4 +83,10 @@ describe('xcode @skip-linux', () => {
     (parseFloat(version)-6.1).should.be.at.least(0);
   });
 
+  it('should get a list of iOS devices', async() => {
+    let devices = await xcode.getConnectedDevices();
+    should.exist(devices);
```

Comment:
actually @sebv I'm a little wary of this. suppose later I rewrote the `getConnectedDevices` function so it uses an npm module instead of calling exec. now the tests have to be changed because the tests are implementation-specific.